temporian/core/operators/window/moving_count.py

```diff
@@ -44,7 +43,7 @@ def feature_schema(self, input: EventSetNode):
 @compile
 def moving_count(
     input: EventSetOrNode,
-    window_length: Duration,
+    window_length: WindowLength,
```

Comment:
What happens if the feature window_length is negative? (e.g., error, skip, replace with zero)

Reply:
outputs missing values in this case, added docs + tests